src/clr-angular/forms/combobox/option.ts

```diff
@@ -1,35 +1,60 @@
 /**
- * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { Component, ElementRef, HostBinding, HostListener, Inject, Input, OnDestroy, Optional } from '@angular/core';
 import { Subscription } from 'rxjs';
 
 import { POPOVER_HOST_ANCHOR } from '../../popover/common/popover-host-anchor.token';
-import { ClrPopoverToggleService } from '../../utils/popover/providers/popover-toggle.service';
 
 import { OptionSelectionService } from './providers/option-selection.service';
-
+import { SelectionModel } from './providers/option-selection.service';
+import { FocusableItem } from '../../utils/focus/focusable-item/focusable-item';
+import { BASIC_FOCUSABLE_ITEM_PROVIDER } from '../../utils/focus/focusable-item/basic-focusable-item.service';
+import { ClrPopoverToggleService } from '../../utils/popover/providers/popover-toggle.service';
 @Component({
   selector: 'clr-option',
-  templateUrl: './option.html',
-  host: { '[class.clr-option]': 'true' },
+  template: `
+    <ng-content></ng-content>
+  `,
+  host: {
+    '[class.clr-option]': 'true',
+    '[attr.role]': '"option"',
+    '[attr.aria-selected]': 'selected',
+  },
+  providers: [BASIC_FOCUSABLE_ITEM_PROVIDER],
 })
 export class ClrOption<T> implements OnDestroy {
-  private subscription: Subscription;
+  private subs: Subscription[] = [];
 
-  @HostBinding('class.active') selected: boolean = false;
+  _selected: boolean = false;
 
   @Input('clrValue') value: T;
 
+  set selected(val: boolean) {
+    this._selected = !!val;
+  }
+
+  @HostBinding('class.active')
```

Comment:
For consistency, could this be with the other values in the `host:` object?

Reply:
From what we've previously talked in the team, only static styles go in the `host:` component configuration, while dynamic values we prefer to add as `HostBinding`s.
I don't fully remember the rationale behind it, was it about some productional compilation, AOT, or just preferred style, but I'm sure we didn't want to have dynamic bindings up in the `host:` config.

Reply:
Living example!
By your request I changed `selectionService.multi` to `selectionService.multiselectable`.
But inside `options.ts` we have this inside the `host:` section
```'[attr.aria-multiselectable]': 'optionSelectionService.multiselectable'```
No refactoring caught it, no IDE caught it, it only appeared during build time.